### PR TITLE
Created composer package + some small fixes

### PR DESCRIPTION
**Pull request by kristoftorfs**

### CODE REVIEW

File: Adminomatic/AutoMapper/Map.php [hunk 1]
```diff
@@ -3,13 +3,13 @@
 namespace Adminomatic\AutoMapper {
 
 	class Map {
-		/** @var Adminomatic\AutoMapper\Property */
+		/** @var Property */
```
Comment (Nylle) [inline Adminomatic/AutoMapper/Map.php]: Is there are reason to remove the fully qualified class name?

Comment (kristoftorfs) [inline Adminomatic/AutoMapper/Map.php]: It's not actually necessary as they are in the same namespace. On top of that my IDE (PHPStorm) marks this as an error when not prepended with a backslash e.g. `\Adminomatic\AutoMapper\Property`.

Comment (Nylle) [inline Adminomatic/AutoMapper/Map.php]: Back in the day when I built this, my IDE (NetBeans) complained about not fully qualified class names, hence I did it like it is. But I didn't to PHP for years now and if you say it's not necessary within the same namespace I'm fine with that.

**Comment by Nylle** [comment]
Generally I'm fine with the changes, but what about the Travis CI issue?

**Comment by kristoftorfs** [comment]
No test classes have been written, and Travis has not been configured for this library. The build fails because it is searching for a Ruby makefile. I didn't really understand why Travis was enabled here in the first place?

**Comment by Nylle** [comment]
Ja, sorry, I think I tried to set it up a long time ago, failed and forgot about it ;-)

**Comment by kristoftorfs** [comment]
Sounds a lot like my first experiences with CI... :-)

**Pull request merged**